ci/lint.sh

```diff
@@ -20,10 +20,10 @@ if [ "$LINT" ]; then
     echo "Linting *.py DONE"
 
     echo "Linting *.pyx"
-    for path in 'window.pyx' "src/join.pyx"
+    for path in 'algos.pyx' 'hashtable.pyx' 'index.pyx' 'lib.pyx' 'parser.pyx' 'tslib.pyx' 'window.pyx' 'src' 'io' 'msgpack'
     do
         echo "linting -> pandas/$path"
-        flake8 pandas/$path --filename '*.pyx' --select=E501,E302,E203,E226,E111,E114,E221,E303,E128,E231,E126
+        flake8 pandas/$path --filename '*.pyx' --select=E501,E302,E203,E111,E114,E221,E303,E128,E231,E126
```

Comment:
change this to not use a loop, IOW ``flake8 pandas --filename '*.pyx' --select=.....`` will catch all ``.pyx`` (even if we add/move them)